include/glow/Base/Type.h

```diff
@@ -60,17 +60,23 @@ static_assert(sizeof(float16_t) == 2, "Half precision should be 16-bit");
 using ShapeVector = llvm::SmallVector<dim_t, max_tensor_dimensions>;
 
 struct ShapeNHWC {
+
+  static constexpr size_t dimN = 0;
```

Comment:
Minor nit: Why not just define an enum for these `dimX` constants?

Reply:
Like this?
```
struct ShapeNHWC {

  enum {
    dimN = 0,
    dimH = 1,
    dimW = 2,
    dimC = 3,
  };
```

Reply:
Yes, exactly! I think you don't even need to explicitly assign the values to them as enum declaration does exactly this anyways. Also, as a very tiny nit, enum cases are usually starting with an upper-case latter.

Reply:
Oh boy ... I'll need to modify in a lot of places. Just to confirm before I embark in this journey: you mean `DimN` right?

Reply:
Yes, I meant `DimN` ;-) But I don't insist on this renaming. It is really a tiny nit. If it is too much work, ignore this renaming part.

Reply:
Done.